src/js/editor.js

```diff
@@ -217,6 +217,8 @@ class ToastUIEditor {
 
     this.setValue(this.options.initialValue, false);
 
+    this.setPlaceholder(this.options.placeholder);
```

Comment:
Please add a comment to the above JS Doc comment above for `placeholder`. The proper position is 130 line that is below the comment of `customConvertor`.